ui/app/components/clients/page/acme.hbs

```diff
@@ -44,46 +44,54 @@
     </:chart>
 
     <:emptyState>
-      <EmptyState
-        @title="No monthly ACME clients"
-        @subTitle="There is no ACME client data available for this date range."
-        @bottomBorder={{true}}
-      />
+      <EmptyState @title="No monthly ACME clients" @message="There is no ACME client data available for this date range." />
     </:emptyState>
   </Clients::ChartContainer>
 
-  <Clients::ChartContainer
-    @title="Monthly new"
-    @description="ACME clients which interacted with Vault for the first time each month. Each bar represents the total new ACME clients for that month."
-    @timestamp={{@activity.responseTimestamp}}
-    @hasChartData={{this.byMonthNewClients}}
-    class="no-legend"
-    data-test-chart="monthly new"
-  >
-    <:stats>
-      <StatText
-        @label="Average new ACME clients per month"
-        @value={{this.average this.byMonthNewClients "acme_clients"}}
-        @size="m"
-        class="chart-subTitle has-top-padding-l"
-      />
-    </:stats>
+  {{#if this.byMonthActivityData}}
+    <Clients::ChartContainer
+      @title="Monthly new"
+      @description="ACME clients which interacted with Vault for the first time each month. Each bar represents the total new ACME clients for that month."
+      @timestamp={{@activity.responseTimestamp}}
+      @hasChartData={{this.byMonthNewClients}}
+      class="no-legend"
+      data-test-chart="monthly new"
+    >
+      <:stats>
+        <StatText
+          @label="Average new ACME clients per month"
+          @value={{this.average this.byMonthNewClients "acme_clients"}}
+          @size="m"
+          class="chart-subTitle has-top-padding-l"
+        />
+      </:stats>
 
-    <:chart>
-      <Clients::Charts::VerticalBarBasic
-        @chartTitle="Monthly new ACME clients"
-        @data={{this.byMonthNewClients}}
-        @dataKey="acme_clients"
-        @chartHeight={{200}}
-      />
-    </:chart>
-  </Clients::ChartContainer>
+      <:chart>
+        <Clients::Charts::VerticalBarBasic
+          @chartTitle="Monthly new ACME clients"
+          @data={{this.byMonthNewClients}}
+          @dataKey="acme_clients"
+          @chartHeight={{200}}
+        />
+      </:chart>
+    </Clients::ChartContainer>
+  {{/if}}
 {{else}}
```

Comment:
I slightly prefer using `{{else if (...)}}` blocks in the template instead of nested `{{#if}}` blocks

Reply:
assuming I'm reading this diff correctly 😄

Reply:
There are two empty states that are different depending on whether or not the data is a date range. 

But, I could rearrange this to make it clearer and instead of using the empty state above (yielded by `ChartContainer`) I can consolidate and reuse the same empty state and just change the copy slightly depending on whether it's a date range or month!